src/cascadia/WpfTerminalControl/WpfTerminalControl.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     
-    <TargetFrameworks>net472;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net6.0-windows</TargetFrameworks>

```

Comment:
Ahh, this project must continue to build for `net472`, because it is being used by Visual Studio for their Terminal implementation and they are using .NET Framework.